docs/rdd-programming-guide.md

```diff
@@ -86,11 +86,9 @@ import org.apache.spark.SparkConf;
 
 <div data-lang="python"  markdown="1">
 
-Spark {{site.SPARK_VERSION}} works with Python 2.6+ or Python 3.4+. It can use the standard CPython interpreter,
+Spark {{site.SPARK_VERSION}} works with Python 2.7+ or Python 3.4+. It can use the standard CPython interpreter,
```

Comment:
That's fine. To further reduce doc update dependencies, maybe just say it works with Python 2 or 3.